README.md

```diff
@@ -31,8 +31,8 @@ when building Symfony full stack applications.
 
 ## Requirements
 
-* PHP 7.1 / 7.2
-* Symfony 2.8 / 3.3 / 3.4 / 4.0
+* PHP 7.1 / 7.2 / 7.2
+* Symfony 3.3 / 3.4 / 4.0 / 5.0
```

Comment:
for symfony 4, we only allow 4.3 / 4.4 - lets just delete the line and refer to composer.json, that will be easier to maintain.